Meadowlark-js/tests/e2e/scenarios/SecurityValidation.test.ts

```diff
@@ -0,0 +1,372 @@
+import { Credentials, createClient } from '../helpers/Credentials';
+import { createResource, deleteResourceByLocation } from '../helpers/Resources';
+import { baseURLRequest, rootURLRequest } from '../helpers/Shared';
+
+describe('given the existance of two vendors and one host', () => {
+  let vendor1Data: Credentials = {
+    clientName: 'Vendor 1',
+    roles: ['vendor'],
+  };
+  let vendor2Data: Credentials = {
+    clientName: 'Vendor 2',
+    roles: ['vendor'],
+  };
+  let host1Data: Credentials = {
+    clientName: 'host 1',
+    roles: ['host'],
+  };
+
+  let vendor1DataAccessToken = '';
+  let vendor2DataAccessToken = '';
+  let host1DataAccessToken = '';
+
+  let contentClassDescriptorLocation = '';
+
+  let educationContentLocation = '';
+  const educationContentEndpoint = 'EducationContents';
+  const educationContentBody = {
+    contentIdentifier: '933zsd4350',
+    namespace: '43210',
+    shortDescription: 'abc',
+    contentClassDescriptor: 'uri://ed-fi.org/ContentClassDescriptor#Presentation',
+    learningResourceMetadataURI: '21430',
+  };
+
+  beforeAll(async () => {
+    vendor1Data = await createClient(vendor1Data);
+    vendor2Data = await createClient(vendor2Data);
+    host1Data = await createClient(host1Data);
+
+    // Vendor 1
+    await baseURLRequest()
+      .post('/oauth/token')
+      .send({
+        grant_type: 'client_credentials',
+        client_id: vendor1Data.key,
+        client_secret: vendor1Data.secret,
+      })
+      .expect(200)
+      .then((response) => {
+        vendor1DataAccessToken = response.body.access_token;
+      });
+
+    // Vendor 2
+    await baseURLRequest()
+      .post('/oauth/token')
+      .send({
+        grant_type: 'client_credentials',
+        client_id: vendor2Data.key,
+        client_secret: vendor2Data.secret,
+      })
+      .expect(200)
+      .then((response) => {
+        vendor2DataAccessToken = response.body.access_token;
+      });
+
+    // Host 1
+    await baseURLRequest()
+      .post('/oauth/token')
+      .send({
+        grant_type: 'client_credentials',
+        client_id: host1Data.key,
+        client_secret: host1Data.secret,
+      })
+      .expect(200)
+      .then((response) => {
+        host1DataAccessToken = response.body.access_token;
+      });
+
+    contentClassDescriptorLocation = await createResource({
+      endpoint: 'contentClassDescriptors',
+      body: {
+        codeValue: 'Presentation',
+        shortDescription: 'Presentation',
+        description: 'Presentation',
+        namespace: 'uri://ed-fi.org/ContentClassDescriptor',
+      },
+      accessToken: vendor1DataAccessToken,
+    });
+
+    expect(contentClassDescriptorLocation).toBeTruthy();
+  });
+
+  afterAll(async () => {
+    await deleteResourceByLocation(contentClassDescriptorLocation);
+  });
+
+  // GET
+  describe('given that Vendor 1 creates an educationContent', () => {
```

Comment:
We are using `EducationContent` here, but that is not the point of the test. Similarly, the test uses "vendor 1" but there is nothing special about that vendor, or about the `EducationContent` resource type. Therefore I recommend simplifying this description:
 ```suggestion
  describe('given a vendor API client has created a resource', () => {
```